### PR TITLE
[k8s-extension] Update extension CLI to v1.6.0

### PR DESCRIPTION
**Pull request by bavneetsingh16**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
k8s-extension

File: src/k8s-extension/setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 # TODO: Add any additional SDK dependencies here
 DEPENDENCIES = []
 
-VERSION = "1.5.3"
+VERSION = "1.5.4"
```
Comment (zhoxing-ms) [inline src/k8s-extension/setup.py:36]: Could you follow the [version guideline](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md) to upgrade the extension version

File: src/k8s-extension/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+1.5.4
+++++++++++++++++++
+* AAD related changes in dataprotection aks ext CLI
+* microsoft.azuremonitor.containers: Make containerlogv2 as default as true and remove region dependency for ARC
+* microsoft.workloadiam: Refactor subcommand invocation
```
Comment (zhoxing-ms) [inline src/k8s-extension/HISTORY.rst:10]: Could you add some tests for these PR changes?

File: src/k8s-extension/setup.py (hunk 1 above)
Comment (bavneetsingh16) [reply src/k8s-extension/setup.py]: done

File: src/k8s-extension/HISTORY.rst (hunk 2 above)
Comment (bavneetsingh16) [reply src/k8s-extension/HISTORY.rst:10]: We have our own custom tests(CI checks) in our forked repo that create a kubernetes cluster and perform create, update, list, show and delete actions for each of the partner extensions.

[https://github.com/AzureArcForKubernetes/azure-cli-extensions/tree/k8s-extension/public/testing/test/extensions/public](https://github.com/AzureArcForKubernetes/azure-cli-extensions/tree/k8s-extension/public/testing/test/extensions/public)

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**